examples/analytics-notification.amp.html

```diff
@@ -66,8 +66,8 @@
   <amp-user-notification
       layout=nodisplay
       id="amp-user-notification1"
-      data-show-if-href="http://localhost:8000/api/show?timestamp=TIMESTAMP"
```

Comment:
so not sure about doing relative paths on these because as examples they were supposed to convey doing requests to a different domain (granted we didn't even do that, we should have maybe spun up a new local server with a different port)

/cc @cramforce